subsys/bluetooth/controller/ll_sw/ull_llcp_remote.c

```diff
@@ -451,7 +451,7 @@ void ull_cp_priv_rr_new(struct ull_cp_conn *conn, struct node_rx_pdu *rx)
 {
 	struct proc_ctx *ctx;
 	struct pdu_data *pdu;
-	uint8_t proc;
+	uint8_t proc = 0;
```

Comment:
how about using PROC_UNKNOWN ?   either here or in default case ?

Reply:
Agree with @sjanc, use PROC_UNKNOWN